src/sampler/glocal.jl

```diff
@@ -47,20 +47,14 @@ function Distributions._rand!(rng::AbstractRNG, d::Glocal, x::AbstractMatrix)
     return x
 end
 
-"""
-    ScaledLogNormal
-
-A [`Glocal`](@ref) distribution `ϵ = α×η` where `α` is uniform and `η` is LogNormal.
-"""
-const ScaledLogNormal = Glocal{Uniform{Float64},MvLogNormal}
```

Comment:
Note that I removed those type declarations, since it created more trouble than not.